code-challenges.test.js

```diff
@@ -94,8 +229,35 @@ var testString2 = "jabberwocky"
 
 // a) Create a test with an expect statement using the variables provided.
 
+
+//??? i tried a few different ways of doing this but i kept getting TypeError: array2 is not iterable 
+//??? the finction works I just could not get it to test. 
+
+// describe("when onlyOne is called", () =>{
+//     it("if argument is testArray1 and testArray2, return [3, 7, 10, 5, 4, 8, 2, 1]",() =>{
+//         const argument = (testArray1 && testArray2);
```

Comment:
testArray1 and testArray2 need to provided to the function as separate arguments. Using && will not join them but create a trurthy/falsy evaluation

Reply:
you can create separate variables like argument1 and argument2